integration_tests/src/main/python/orc_write_test.py

```diff
@@ -98,6 +101,23 @@ def test_write_round_trip(spark_tmp_path, orc_gens, orc_impl):
             data_path,
             conf={'spark.sql.orc.impl': orc_impl, 'spark.rapids.sql.format.orc.write.enabled': True})
 
+@pytest.mark.parametrize('orc_gen', [int_gen], ids=idfn)
+@pytest.mark.parametrize('orc_impl', ["native", "hive"])
+def test_write_with_stripe_size_rows(spark_tmp_path, orc_gen, orc_impl):
+    gen_list = [('_c0', orc_gen)]
+    data_path = spark_tmp_path + '/ORC_DATA'
+    stripe_size_rows = 10000
```

Comment:
Add comment: cuDF may round the number or elements in each stripe (except the last) to a multiple of 8 to be able to efficiently encode the validity masks.
And use a integer divisible by 8.